### ISSUE TITLE
Tracking Issue: WCAG 2.1 AA Accessibility - Fixing known issues

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#31476]
### Plan
This tracking issue will currently serve as a placeholder. It should be used to track the progress of fixing known accessibility issues in our application

#### Legend

- 👩 Customer issue
- 🐛 Bug
- 🧶 Technical debt
- 🎩 Quality of life
- 🛠️ [Roadmap](https://handbook.sourcegraph.com/departments/product-engineering/process/planning-process#roadmap)
- 🕵️ [Spike](https://en.wikipedia.org/wiki/Spike_(software_development))
- 🔒 Security issue
- 🙆 Stretch goal
